### PR TITLE
Added toolbar with run button.

### PR DESCRIPTION
**Pull request by taier1**

### CODE REVIEW

File: elements/asq-java-q-presenter.html [hunk 1]
```diff
@@ -24,69 +32,126 @@
...
-  #le-content{
-    @apply(--layout-horizontal);
+
+  #statusMsg{
+     padding:0 0.4em;
+     @apply(--layout-flex);
+  }
+
+  #errorMsg{
+    @apply(--asq-sqlite-q-error-message);
```
Comment (BalestraPatrick) [inline elements/asq-java-q-presenter.html]: Should replace `sqlite` with `java`

**Review by BalestraPatrick — CHANGES_REQUESTED** [review]

File: asq-java-q.html [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <link rel="import" href="../polymer/polymer.html">
```
Comment (triglian) [inline asq-java-q.html:1]: you still have to figure out if `selectedTab` is used

File: elements/asq-java-q-presenter.html [hunk 3]
```diff
@@ -3,6 +3,14 @@
 <link rel="import" href="../../iron-list/iron-list.html">
```
Comment (triglian) [inline elements/asq-java-q-presenter.html:3]: this element should use the  `ASQ.asqQuestionElementBehavior` like [here]( https://github.com/ASQ-USI-Elements/asq-multi-choice-q/blob/master/elements/asq-multi-choice-q-presenter.html#L121) . The same holds true for the viewer element

**Review by BalestraPatrick — APPROVED** [review]

File: elements/asq-java-q-presenter.html (hunk 3 above)
Comment (BalestraPatrick) [reply elements/asq-java-q-presenter.html:3]: Added it back. Can you please review it again so that we can merge it?

File: asq-java-q.html (hunk 2 above)
Comment (BalestraPatrick) [reply asq-java-q.html:1]: It's not needed at the moment.

File: elements/asq-java-q-viewer.html [hunk 4]
```diff
@@ -166,15 +268,32 @@
        *
        * @method submit
        */
-      submit: function () {
+      submit: function (evt) {
```
Comment (triglian) [inline elements/asq-java-q-viewer.html:249]: Submit should just submit the code as an answer.

File: elements/asq-java-q-viewer.html [hunk 5]
```diff
@@ -155,6 +250,13 @@
       _editorValueChanged: function(evt){
         this.value = evt.target.editorValue;
       },
+
```
Comment (triglian) [inline elements/asq-java-q-viewer.html:237]: remove commented code

**Review by triglian — CHANGES_REQUESTED** [review]

**Pull request closed without merging**